src/js/integrations/reactnativeinfo.ts

```diff
@@ -0,0 +1,64 @@
+import { Context, Event, EventHint, EventProcessor, Integration } from '@sentry/types';
+
+import { isFabricEnabled, isHermesEnabled, isTurboModuleEnabled } from '../utils/architecture';
+import { ReactNativeError } from './debugsymbolicator';
+
+export interface ReactNativeContext extends Context {
+  jsEngine?: string;
+  turboModule: boolean;
+  fabric: boolean;
+  componentStack?: string;
```

Comment:
Id you look at https://develop.sentry.dev/sdk/event-payloads/contexts/
You see that the protocol is snake case.